app/services/idv/agent.rb

```diff
@@ -23,7 +22,7 @@ def proof_resolution(
 
       job_arguments = {
         encrypted_arguments: encrypted_arguments,
-        should_proof_state_id: should_proof_state_id,
+        should_proof_state_id: false,
```

Comment:
worth dropping a comment that this is intended to be removed?

Reply:
Yeah. I'm also going to have to break this pull request into multiple changes since I need to make the changes in the jobs first before making any changes to the args we pass to them.